public/app/routes/routes.tsx

```diff
@@ -30,14 +30,6 @@ export function getAppRoutes(): RouteDescriptor[] {
         () => import(/* webpackChunkName: "DashboardPage" */ '../features/dashboard/containers/DashboardPage')
       ),
     },
-    {
```

Comment:
Think this one needs to be left untouched because `:type` could be scripted and snapshots.

Reply:
I guess you're right, yes, wasn't sure either. Reverting...